_overviews/scala3-book/taste-indentation.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Indentation
+type: section
+description: This section demonstrates Scala 3 Indentation.
+num: 8
+previous-page: taste-vars-data-types
+next-page: taste-control-structures
+---
+
+Indentation refers to the increase or decrease of space between the left and right margin of a paragraph.
+In some programming languages, tab key is used to make indentations. This is not the case in Scala. Tab key automatically creates four spaces which Scala doesn't support.
```

Comment:
This does not seem to be true, according to the language reference: https://docs.scala-lang.org/scala3/reference/other-new-features/indentation.html#spaces-vs-tabs